boards/nxp/frdm_ke17z512/doc/index.rst

```diff
@@ -61,7 +61,8 @@ features:
 +-----------+------------+-------------------------------------+
 | I2C       | on-chip    | I2C                                 |
 +-----------+------------+-------------------------------------+
-
+| RTC       | on-chip    | rtc                                 |
```

Comment:
the interface "right column" is not "rtc", it is "counter". "rtc" is a different driver class, which this is not being supported through

Reply:
Updated